projects/habitat.md

```diff
@@ -18,6 +18,9 @@ Christopher Maier
 
 #### Project Approvers
 
+Salim Alam
```

Comment:
We're doing names alphabetically in these lists. We should probably write that down somewhere.

Reply:
Fixed the ordering